tests/core/mountpoint/test_fuse.py

```diff
@@ -189,3 +190,20 @@ async def _mocked_bootstrap_mountpoint(*args):
 
         # Test is over, stop alice2 mountpoint and exit
         nursery.cancel_scope.cancel()
+
+
+@pytest.mark.linux
+@pytest.mark.mountpoint
+@pytest.mark.parametrize("name", ("x" * 256, "飞" * 85 + "x"))
+def test_create_a_file_with_a_name_exceeding_255_bytes(mountpoint_service, name):
+    """
+    This test is kept as a fuse/linux-only test since accesses to files with name
+    over 255 bytes on windows do not reach the `winfsp_operations` module. Weirdly
+    enough, the version with ascii characters return with an "Invalid argument" error
```

Comment:
why are we checking the same errno code then ?

Reply:
Ah that part is about windows, I'll edit it to make it more clear.

Reply:
65c8edbf94a88609907f7dbec4c55e7dd9c31ca3